src/components/PathwaysList/PathwayModal.tsx

```diff
@@ -49,6 +52,30 @@ const NewPathwayModal: FC<NewPathwayModalProps> = ({ open, onClose }) => {
     [addPathway, closeModal]
   );
 
+  const handleUpdatePathway = useCallback(
+    (event: FormEvent<HTMLFormElement>): void => {
+      event.preventDefault();
+      if (editPath) {
+        if (pathwayNameRef.current?.value) editPath.name = pathwayNameRef.current?.value;
+        if (pathwayDescRef.current?.value !== '')
+          editPath.description = pathwayDescRef.current?.value;
+        const pathwayId = editPath.id;
+        const pathwayIndex = pathways.findIndex(pathway => pathway.id === pathwayId);
+        updatePathwayAtIndex(editPath, pathwayIndex);
+        onClose();
+      }
+    },
+    [pathways, updatePathwayAtIndex, editPath, onClose]
+  );
+
+  let label = 'Pathway Name';
+  let description = 'Pathway Description';
+  if (editPath) {
+    label = editPath.name;
+    const metaDescription = editPath.description;
```

Comment:
As Dylan mentioned we don't want to update the labels here but rather set the values of the refs